views/default/forms/scheduling/days.php

```diff
@@ -2,92 +2,108 @@
 
 $entity = elgg_extract('entity', $vars);
 
+$rows = array();
 $days = $entity->getSlotsGroupedByDays();
-
-$num_rows = $days ? count($days) : 1;
-$num_cols = count(reset($days)) ? count(reset($days)) : 1;
-
-$headings = '<th></th>';
-for ($heading = 0; $heading < $num_cols; $heading++) {
-	$text = elgg_echo('scheduling:slot:title', array($heading));
-	$headings .= "<th>$text</th>";
+$num_columns = 0;
+if (!empty($days)) {
+	foreach ($days as $date => $slots) {
+		foreach ($slots as $slot) {
+			$rows[$date][] = date("H:i", (int) $slot->title);
+		}
+		if (count($slots) > $num_columns) {
```

Comment:
Yes, since we have the last column with actions, we need to make sure the number of table cells in each row is the same.